ddtrace/contrib/pymongo/client.py

```diff
@@ -211,7 +233,7 @@ def write_command(self, *args, **kwargs):
         with self.__trace(cmd) as s:
             result = self.__wrapped__.write_command(*args, **kwargs)
             if result:
-                s.set_metric(mongox.ROWS, result.get("n", -1))
+                s.set_metric(db.ROWCOUNT, result.get("n", -1))
```

Comment:
Is mongox.ROWS used anywhere else? If not, we should deprecate the constant

Reply:
nope, added deprecation